### ISSUE TITLE
"List my network adapters..." - CMD Flashes and Closes

### ISSUE THREAD

**Issue by war59312** [raiguard/ModernGadgets#281]
Hi,

For Network Meter Settings > "List my network adapters..."

It opens a CMD window but then closes. Should add a pause to it so it stays open on screen.

Best,

Will

**Comment by SilverAzide** [comment]
Huh?  It does have a pause in it.  Check the last line of the file `ModernGadgets\@Resources\Addons\NetMonitor\NetworkAdapterList.cmd`.  It sounds like you have a problem someplace. Open a command prompt and run the batch file manually so you can catch the error message.

**Comment by war59312** [comment]
OK, that's weird, if I run the cmd I get:

`This program is blocked by group policy. For more information, contact your system administrator.`

Only thing I have blocked is Chrome's software_reporter_tool.exe via HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer\DisallowRun .

I removed that and did `gpupdate /force` and still blocked.

Seems all .cmd blocked now. This is my personal computer and not sure what blocked.

Ok, it was AppLocker at fault, I was testing it yesterday.

Sorry for false alarm, don't run cmd every day.
